ttnn/cpp/ttnn/operations/conv/conv2d/device/kernels/writer_tiled_out_1d_mcast_sender_conv_weights_tiled_col_to_rm_blocks.cpp

```diff
@@ -203,7 +203,7 @@ void kernel_main() {
                         weights_mcast_dest_noc_end_y,
                         weights_start_address);
                         // num_dests must not include source, since we are NOT really doing a local copy!
-                        noc_async_write_multicast(weights_start_address, weights_multicast_data_addr, weights_block_size_bytes, weights_mcast_num_cores, true, true);
```

Comment:
What does the `linked` argument do? Should it be changed for all the multicasts in conv2d?

Reply:
Can you explain why linked was flipped from false -> true, Djordje Maksimovic and Davor insisted that we need to enable this everywhere to avoid potential hangs?

Reply:
the linked flag here was flipped from true -> false. The flag that always needs to be enabled to avoid hangs is multicast_path_reserve, not linked.

Flipping the linked flag from true to false would be a perf degradation for cases that it previously worked. If this is set to false to make it functional for BH, then it should be gated to BH only to avoid perf degradation on other arches.

Reply:
yeah @tt-aho got this better than me.

Reply:
IIRC, this was required for BH to avoid hangs. If this causes perf degradation on WH/GS, then I will gate it for BH only.

Reply:
Agreed. It'd be good to know perf and adjust the code accordingly.

Also if this causes hangs without a known reason, it would be good to investigate why in a separate issue.